keyboards/skme/zeno/config.h

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2019 Holten Campbell
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x2448 // baul.xyz / sandkeys.me
```

Comment:
This vendor ID is in use. 
https://www.the-sz.com/products/usbid/index.php?v=0x2448&p=&n=

In general, anything about 0x4000 is unused, and shouldn't be used "soon".  But the further past this, the better.